build.gradle

```diff
@@ -54,7 +54,9 @@ dependencies {
     compileOnly group: "io.kestra", name: "core", version: kestraVersion
 
     // libs included in the final jar
-    api group: 'com.google.code.gson', name: 'gson', version: '2.10.1'
+    api group: 'com.surrealdb', name: 'surrealdb-driver', version:  '0.1.0'
+    api group: 'org.java-websocket', name: 'Java-WebSocket', version: '1.3.0'
```

Comment:
This lib didn't seems to be used directly, is it really needed? Isn't it a shadow dependency of the SurrealDB driver?

Reply:
It's should be in shadow dependency, but if not include this lib in dependency then, on gradle assemble it tells me, that claass WebSocketClient not found